docs/source/get-started/package-managers.rst

```diff
@@ -3,20 +3,32 @@ Package Managers
 
 Use your favorite package manager to install Kokkos.
 
-System package managers
+Package managers
 ~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can we add a comment below here:

```rst
.. warning::

   Only the Spack package below is maintained by Kokkos developers and is our only official supported package. Others may be used, but please contact the listed maintainers for any issues.
```

Just to make it extra clear that we can't commit to supporting all the OS packages?